frontend/public/components/dashboard/dashboards-page/overview-dashboard/utilization-card.tsx

```diff
@@ -92,26 +202,51 @@ const UtilizationCard_: React.FC<DashboardItemProps & WithFlagsProps> = ({
         <UtilizationItem
           title="CPU"
           data={cpuStats}
-          error={cpuUtilizationError}
+          error={
+            cpuUtilizationError ||
+            cpuTopConsumersByPodsError ||
```

Comment:
error/loading should be handled inside `PopupComponent`, we can still show Usage even if top consumers queries are failing.